command/pr.go

```diff
@@ -412,28 +412,28 @@ func printPrs(w io.Writer, totalCount int, prs ...api.PullRequest) {
 				var summary string
 				if checks.Failing > 0 {
 					if checks.Failing == checks.Total {
-						summary = utils.Red("All checks failing")
+						summary = utils.Red("× All checks failing ")
 					} else {
-						summary = utils.Red(fmt.Sprintf("%d/%d checks failing", checks.Failing, checks.Total))
+						summary = utils.Red(fmt.Sprintf("× %d/%d checks failing ", checks.Failing, checks.Total))
 					}
 				} else if checks.Pending > 0 {
-					summary = utils.Yellow("Checks pending")
+					summary = utils.Yellow("- Checks pending ")
 				} else if checks.Passing == checks.Total {
-					summary = utils.Green("Checks passing")
+					summary = utils.Green("✓ Checks passing ")
 				}
-				fmt.Fprintf(w, " - %s", summary)
+				fmt.Fprintf(w, "%s", summary)
```

Comment:
Nit: although it's not incorrect, you never really need to use `Fprintf` if the format string is exactly `"%s"`. That format string instructs Fprintf that the string argument should just be printed unchanged, i.e. that there is no specific formatting to apply. If no formatting is needed, then it's better to use the regular `Fprint`
```suggestion
				fmt.Fprint(w, summary)
```

Reply:
There actually should be a ` - ` here, I missed it! 🤦‍♀